### ISSUE TITLE
SignalR integration with asp.net core

### ISSUE THREAD

**Issue by mohammad-shadmehr** [aspnetboilerplate/module-zero-core-template#186]
Hi,
I am trying to integrate SignalR to my ASP.ZERO template by following instruction in the document and some tickets in the forum with that regard ([https://forum.aspnetboilerplate.com/viewtopic.php?f=5&t=10737&p=26657&hilit=%40aspnet%2Fsignalr#p26657])

The latest error that I am dealing with is within the SignalRAspNetCoreHelper.ts file as:
> ERROR in src/shared/helpers/SignalRAspNetCoreHelper.ts(13,13): error TS2322: Type '{ autoConnect: true; connect: undefined; hubs: undefined; qs: string; url: string; }' is not assignable to type 'typeof signalr'.
  Object literal may only specify known properties, and 'qs' does not exist in type 'typeof signalr'.

Any thought? In overall I believe that the documents need to include more details specially when it comes adding a feature which does not exist with the template at time of its creation.

Thanks ;)

**Comment by acjh** [comment]
Update `abp-web-resources` package:

https://github.com/aspnetboilerplate/module-zero-core-template/blob/620a99d0213df160ef3b60505f7bb135d0bc2ccb/angular/package.json#L36

**Comment by mohammad-shadmehr** [comment]
Hi Aaron,
I have successfully added SignalR to my asp core solution (ASP Zero). For some reason the notification is not real time and I have to still refresh the page to get the notification bell jingling! 
In the premium forum I noticed that you have indicated that this is a known issue raised before but as my subscription was finished I cannot reach to the issue on github. 
All I have is a button on my Angular page which would call a service to publish a notification! in the same page where the button is I have defined a function to register to a notification:
registerEvents(): void { abp.event.on('abp.notifications.received', function (userNotification) { alert(userNotification); }); }
seems the following trigger within the abp.signalr-client.j is get fired at all:
//Register to get notifications connection.on('getNotification', function (notification) { abp.event.trigger('abp.notifications.received', notification); });

thanks
